pyproject.toml

```diff
@@ -82,6 +83,7 @@ job-queue = [
 ]
 passport = [
     "cryptography!=3.4,!=3.4.1,!=3.4.2,!=3.4.3,>=39.0.1",
+    "cffi >= 1.17.0rc1; python_version > '3.12'"
```

Comment:
can you please add a comment aboud cffi being a dependency of crypto that added 3.13 support only in 1.17.0rc1?